data/management/commands/matchsubmissions.py

```diff
@@ -0,0 +1,22 @@
+import logging
+import time
+
+from django.core.management.base import BaseCommand
+
+from matcher.tasks import match_exercise
+from data.models import Course
+
+
+class Command(BaseCommand):
+    help = "Calculates remaining matches for an exercise."
```

Comment:
"Calculate" is a misleading word choice here. Is "Runs remaining matches..." more descriptive?

Reply:
Hmm.. "running" the matches is very wake for me as this is pure calculation, should perhaps be: Calculates remaining submission similarities for an exercise.

Reply:
Ok, it could stay as "calculate". The text could be changed so that it is more clear that the command runs the similarity comparisons for the not-yet-matched submissions in the exercise.

Reply:
Good! :+1: